cds-snc/covid-alert-portal
# Build "register" page, confirmation page, and "registrant" model

# Summary | Résumé

Added a new "register" app, a new "Registrant" model, with a start page and a confirmation page.

- add start page
- add confirmation page
- add models for
  - Registrant details
- add page-by-page flow for registering a person (email and name) 

page-by-page flow includes:

1. add email
2. add name

Things we are skipping for this story: 
- Email confirmation
- Adding a location

Also, added a feature flag so that we can ship this to prod without it being visible.

## How to test this feature

- If you build it locally, make sure the `QR_CODES` waffle switch is active. 
- Go to url "/register"
- Click start
- Put in an email address
- Put in a name
- Confirmed!

## gif

![Screen Recording 2021-01-29 at 10 04 06 AM](https://user-images.githubusercontent.com/2454380/106294244-ef665700-621c-11eb-9214-c52392cb9c07.gif)
